lib/shopify-cli/core/entry_point.rb

```diff
@@ -11,6 +11,8 @@ def call(args, ctx = Context.new)
             IO.open(9) { is_shell_shim = true }
           rescue Errno::EBADF
             # This is expected if the descriptor doesn't exist
+          rescue ArgumentError
```

Comment:
It's low risk, but we might want to check for a specific message here before suppressing it, just in case there could be some other thing that raises an `ArgumentError`.